connaisseur/util.py

```diff
@@ -130,3 +131,8 @@ def get_kube_version():
     regex = r"v(\d)\.(\d{1,2})\.(.*)"
     match = re.match(regex, version)
     return match.groups() if match else ("0", "0", "0")
+
+
+def feature_flag_on(flag: str, default: bool = False):
```

Comment:
Maybe this is a nice time to make defaults a single thing. I think we're sure that the env var must exist due to defaults in `helm/templates/env.yaml`, right? Otherwise we could also not set defaults in Helm and instead have a feature_flag.py (or the like), which would get the feature value from environment OR the appropriate default. 

Both solutions would prevent multiple instances of defaults being spread around the code, which could hamper development if we changed it in the future